dashboard/app/controllers/api/v1/schools_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::SchoolsController < ApplicationController
+  # GET /api/v1/school/<school_district_id>/<school_type>
+  def index
+    schools = School.where(school_district_id: params[:school_district_id], school_type: params[:school_type]).pluck(:id, :name)
+    render json: ActiveModel::Serializer.new(schools, each_serializer: Api::V1::SchoolSerializer)
```

Comment:
I had to also remove `pluck` for this to work. Once I did that, it picked up the list of attributes specified in the serializer, which previously it was ignoring (hence me adding the pluck). This way seems good.